samples/sample_vpp/include/sample_vpp_utils.h

```diff
@@ -462,47 +462,21 @@ struct sAppResources
     GeneralWriter*      pDstFileWriters;
     mfxU32              dstFileWritersN;
 
-    sFrameProcessor*    pProcessor;
-    mfxVideoParam*      pVppParams;
-    sMemoryAllocator*   pAllocator;
-    sInputParams*       pParams;
-    SurfaceVPPStore*    pSurfStore;
-
-    /* VPP extension */
-    mfxExtVppAuxData*   pExtVPPAuxData;
-    mfxExtVPPDoUse      extDoUse;
-    mfxU32              tabDoUseAlg[ENH_FILTERS_COUNT];
-    mfxExtBuffer*       pExtBuf[1+ENH_FILTERS_COUNT];
-
-    /* config video enhancement algorithms */
-    mfxExtVPPProcAmp    procampConfig;
-    mfxExtVPPDetail     detailConfig;
-    mfxExtVPPDenoise    denoiseConfig;
-#ifdef ENABLE_MCTF
-    mfxExtVppMctf       mctfConfig;
-#endif
-    mfxExtVPPRotation   rotationConfig;
-    mfxExtVPPScaling    scalingConfig;
-#if MFX_VERSION >= 1025
-    mfxExtColorConversion    chromaSitingConfig;
-#endif
-    mfxExtVPPFrameRateConversion    frcConfig;
-    mfxExtVPPDeinterlacing deinterlaceConfig;
-    mfxExtVPPVideoSignalInfo  videoSignalInfoConfig;
-    mfxExtVPPMirroring  mirroringConfig;
-    mfxExtVPPComposite     compositeConfig;
-
+    sFrameProcessor*       pProcessor;
+    MfxVideoParamsWrapper* pVppParams;
+    sMemoryAllocator*      pAllocator;
+    sInputParams*          pParams;
+    SurfaceVPPStore*       pSurfStore;
+
+    // number of video enhancement filters (denoise, procamp, detail, video_analysis, multi_view, ste, istab, tcc, ace, svc)
+    constexpr static uint32_t ENH_FILTERS_COUNT = 20;
```

Comment:
this hardcoded value makes we wonder when we will crash

Reply:
moved from
https://github.com/Intel-Media-SDK/MediaSDK/blob/b8ac0a6bb6c8d32f08066b765dda27bd8a09be17/samples/sample_vpp/include/sample_vpp_config.h#L46